docs/swagger.yml

```diff
@@ -553,13 +553,6 @@ definitions:
         description: Most recent time that app was updated. Always in UTC.
         readOnly: true
 
-  Version:
-    type: object
-    properties:
-      version:
-        type: string
-        readOnly: true
-
   RoutesWrapper:
```

Comment:
Hmm, why did this get removed?

Reply:
this was from the merge commit. rebase attempts were not fun, sorry